install-deps.sh

```diff
@@ -167,6 +167,8 @@ function activate_virtualenv() {
     . $env_dir/bin/activate
 }
 
+test "$1" = "skip-virtualenv" && exit 0
```

Comment:
The tests are run via tox which needs virtualenv. The other reason is to be able to run the tests without network access. a) run install-deps.sh with network access to pull dependencies (packages + python) and do nothing else, b) run make check with no network access